apps/robot-shop/manifest.yaml

```diff
@@ -206,7 +206,6 @@ metadata:
   name: mongodb-volume-claim
   namespace: robot-shop
 spec:
-  storageClassName: ""
```

Comment:
Won't this still auto-provision with the default storage account?

Reply:
@shawn-hurley we need to use default because we are using this app at various different places, having a hard-coded sc will break other deployments. I do see value in having the build script modify this value just like how it's done for `v1beta1` for deployment.

Reply:
Actually, I didn't realize it was an empty string. That was there to use NFS. Ignore my previous comment.

Reply:
It does not use the default when you set it with an empty string. At least not on ROKS. If the value is absent altogether it uses default.